readme.md

```diff
@@ -28,7 +28,10 @@ The benchmark suite requires a storage layer, distributed (such as HDFS/S3/Azure
 	- Compute Layer:
 		 - When running with local spark, the `spark` configuration section in config.yml should work out of the box.
 		 - When running on a cluster manager, the cluster first needs to be installed and configured. If you use YARN, [this](https://hadoop.apache.org/docs/current/hadoop-yarn/hadoop-yarn-site/YARN.html#:~:text=core%2Ddefault.xml-,Apache%20Hadoop%20YARN,or%20a%20DAG%20of%20jobs.) and [this](http://spark.apache.org/docs/latest/running-on-yarn.html) are good places to start. Once that is done, the `spark` and `hadoop` configuration sections need to be changed to point to the cluster manager.
-	 - We recommend setting `hadoop.tmp.dir` to a fast SSD partition for each machine. It is set to a subfolder in `/scratch` by default.
+	 - Ephemeral Disks
+		 - We recommend setting `hadoop.tmp.dir` to a fast SSD drive for each machine. It is set to a subfolder in `/scratch` by default.
+		 - On AWS, we typically use m5d/r5d instance types, which already come with  NVMe SSD ephemeral disks, but are not mounted anywhere. We use this [script](https://gist.github.com/rahij/c7e84c5e0ec06e025873b09882123654) to mount it to `scratch`. These already come with hardware level encryption, so no LUKS encryption is necessary.
```

Comment:
let's just put the scripts in this repo? like, make a subfolder called `scripts/` or something?